src/io/flutter/run/daemon/FlutterAppListener.java

```diff
@@ -105,9 +105,19 @@ public void onAppLog(@NotNull DaemonEvent.AppLog message) {
   @Override
   public void onAppProgressStarting(@NotNull DaemonEvent.AppProgress event) {
     progress.start(event.message);
+
     if (event.getType().startsWith("hot.")) {
+      // We clear the console view in order to help indicate that a reload is happening.
```

Comment:
Okay, yeah, this seems fine; if we're debugging we probably want the verbose log anyway.